src/host/renderData.cpp

```diff
@@ -110,8 +110,6 @@ void RenderData::UnlockConsole() noexcept
 const TextAttribute RenderData::GetDefaultBrushColors() noexcept
 {
     const CONSOLE_INFORMATION& gci = ServiceLocator::LocateGlobals().getConsoleInformation();
-    _defaultForeground = gci.GetDefaultForeground();
-    _defaultBackground = gci.GetDefaultBackground();

```

Comment:
These values were just being "cached" here to optimise the `LookupAttributeColors` call below, but that's no longer needed now that the default index positions are essentially precalculated at startup.